lib/stm32/common/usart_common_f03.c

```diff
@@ -0,0 +1,157 @@
+/** @addtogroup usart_file
+
+ @author @htmlonly &copy; @endhtmlonly 2016 Cem Basoglu <cem.basoglu@web.de>
+
+ */
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2016 Cem Basoglu <cem.basoglu@web.de
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**@{*/
+
+#include <libopencm3/stm32/usart.h>
+
+/*---------------------------------------------------------------------------*/
+/** @brief USART enable data inversion
+
+ Logical data from the data register are send/received in negative/inverse logic. (1=L, 0=H). The parity bit is also inverted.
+
+ @This bit field can only be written when the USART is disabled.
+
+ @param[in] usart unsigned 32 bit. USART block register address base @ref
```

Comment:
no need for "unsigned 32 bit" that's conveyed alreay by the function signatures.